tests/utils.py

```diff
@@ -125,8 +136,6 @@ def init_position_scalars(ds):
         if v in ds.data_vars:
             ds = ds.drop(v)
 
-    xy_periodic = ds.xy_periodic
     ds = xr.merge([ds, ds_position_scalars])
-    ds.attrs["xy_periodic"] = xy_periodic
```

Comment:
I guess this isn't needed anymore because we explicitly indicate whether the domain is xy-periodic when calling `advtraj.integrate_trajectories`, is that why you're removing this? I think I must've added this when I was developing the code and didn't realise while changing things around that it wasn't needed anymore.

Reply:
Basically, except in test/utils.py above, "xy_periodic" is an attribute everywhere. (I plan to add it to my 'MONC io" grid correction code). You needed this here to convert the class attribute to a member of attr. It you set it to an attribute in the first place it gets carried over. I think it's better from a NetCDF point of view to stuff it in attrs, but I'm not very NetCDF savvy.